tests/data/test_alpaca.py

```diff
@@ -6,7 +6,7 @@ def test_alpaca(mock_tockenizer, alpaca_path):
     alpaca = Alpaca(
         test_split_fraction=0.5,
         download_dir=alpaca_path.parent,
-        data_file_name=alpaca_path.name,
```

Comment:
renamed these because it through the CLI, it feels redundant to write data.data.